source/_integrations/airvisual.markdown

```diff
@@ -37,74 +36,82 @@ api_key:
   description: Your AirVisual API key.
   required: true
   type: string
-monitored_conditions:
-  description: "The air quality standard(s) to use (`us` for U.S., `cn` for Chinese)."
-  required: true
-  type: list
-  default: ['us', 'cn']
+geographies:
```

Comment:
I'm not quite sure how to represent and document the schema used here:

```python
GEOGRAPHY_COORDINATES_SCHEMA = vol.Schema(
    {
        vol.Required(CONF_LATITUDE): cv.latitude,
        vol.Required(CONF_LONGITUDE): cv.longitude,
    }
)

GEOGRAPHY_PLACE_SCHEMA = vol.Schema(
    {
        vol.Required(CONF_CITY): cv.string,
        vol.Required(CONF_STATE): cv.string,
        vol.Required(CONF_COUNTRY): cv.string,
    }
)

CLOUD_API_SCHEMA = vol.Schema(
    {
        vol.Required(CONF_API_KEY): cv.string,
        vol.Optional(CONF_GEOGRAPHIES, default=[]): vol.All(
            cv.ensure_list,
            [vol.Any(GEOGRAPHY_COORDINATES_SCHEMA, GEOGRAPHY_PLACE_SCHEMA)],
        ),
        vol.Optional(CONF_SHOW_ON_MAP, default=True): cv.boolean,
    }
)

CONFIG_SCHEMA = vol.Schema({DOMAIN: CLOUD_API_SCHEMA}, extra=vol.ALLOW_EXTRA,)
```

Reply:
This is hard to format indeed.

I think your current listing is fine, some examples might be the key here. (which you already added 👍 )